### PR TITLE
Changes in types and messages

### PR DESCRIPTION
**Pull request by gzukowski**
**What?**  
Changes in types and messages.

**Why?**  
There is a need to change the types for "Instance", because of the required content.

**Usage:**

- 
-
-

**Clickup Task:** 

**How it works:**

-
-
-

**Review checks:**

These aspects need to be checked by the reviewer:

- [ ] Verify and confirm operation (please post a screenshot) 
- [ ] All STH tests pass
- [ ] All [Scramjet Cloud Platform](https://docs.scramjet.org/platform) tests pass
- [ ] Documentation is updated or no changes

### CODE REVIEW

File: packages/types/src/rest-api-manager/get-topics.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
 export type GetTopicsResponse = {
     name: string;
     contentType: string;
+    actors: {
+        hostId?:string;
```
Comment (ErykSol) [inline packages/types/src/rest-api-manager/get-topics.ts:5]: Missing fields in response.

File: packages/types/src/instance-store.ts [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-export type InstanceArgs = any[];
+import { InstanceArgs, InstanceId, InstanceStatus } from "./instance";
+import { SequenceInfoInstance } from "./sequence-adapter";
 
 export type Instance = {
     id: InstanceId,
     appConfig?: AppConfig,
     args?: InstanceArgs,
     provides?: string,
     requires?: string,
```
Comment (ErykSol) [inline packages/types/src/instance-store.ts:10]: requires- redundant field? Check if it is used somewhere.

File: packages/types/src/instance-store.ts [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
-    sequence: string,
-    ports?: Record<string, number>
+    sequenceInfo: SequenceInfoInstance,
+    ports?: Record<string, number>,
```
Comment (ErykSol) [inline packages/types/src/instance-store.ts:12]: ports- check if needed.

File: packages/types/src/instance-store.ts [hunk 4]
```diff
@@ -1,16 +1,15 @@
...
     created?: Date,
     started?: Date,
     ended?: Date,
```
Comment (ErykSol) [inline packages/types/src/instance-store.ts:15]: missing field status- check with fronend-team what values are used

File: packages/types/src/rest-api-manager/get-list.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { ConnectedSTHInfoDetails } from "./common";
+
+export type GetListResponse = {
+      id: string,
+      info: ConnectedSTHInfoDetails,
+      healthy: boolean,
+      selfHosted: boolean,
+      isConnectionActive: boolean,
+      description?: string,
+      tags?: Array<string>,
```
Comment (ErykSol) [inline packages/types/src/rest-api-manager/get-list.ts:10]: string[] -cleaner code

**Review by ErykSol — CHANGES_REQUESTED** [review]

**Comment by ErykSol** [comment]
Missing openApi docs update

**Review by ErykSol — APPROVED** [review]

**Comment by a-tylenda** [comment]
Merge when openApi docs is done by @gzukowski

**Pull request merged**